client/src/Components/StrangePoison/StrangePoison.jsx

```diff
@@ -0,0 +1,130 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import SPListItem from './SPListItem';
+import SPListNewItem from './SPListNewItem';
+class StrangePoison extends Component {
+  static propTypes = {
+    List: PropTypes.object.isRequired,
+  };
+  render() {
+    const {itemsSP} = this.props.List;
```

Comment:
thống nhất lại 1 chút nhé.
lúc thì e gọi kiểu này.
lúc thì lại dùng luôn`this.props` ở render.
Nên thống nhất 1 kiểu thôi. Anh nghĩ là dùng kiểu này thì tốt hơn.